docs/contracts.rst

```diff
@@ -556,13 +559,21 @@ In particular, the following operations will consume more gas than the stipend p
 
 Please ensure you test your fallback function thoroughly to ensure the execution cost is less than 2300 gas before deploying a contract.
 
+.. note::
+    Even though the fallback function cannot have arguments, one can still use ``msg.data`` to retrieve
+    any payload supplied with the call.
+
 .. warning::
     Contracts that receive Ether directly (without a function call, i.e. using ``send`` or ``transfer``)
     but do not define a fallback function
     throw an exception, sending back the Ether (this was different
     before Solidity v0.4.0). So if you want your contract to receive Ether,
     you have to implement a fallback function.
 
+.. warning::
+    A contract without a payable fallback function can receive Ether as a recipient of a `coinbase transaction` or
```

Comment:
double ```. Also perhaps add `(miner block reward)`

Reply:
Perhaps also add a sentence: A contract cannot react to such Ether transfers and thus also cannot reject them. This means that a contract's balance can be higher than the sum of a manually added "internal accounting".

Reply:
It is single "`" on purpose, because that is italic.